wsr-tic-tac-toe/src/__tests__/game-board.test.tsx

```diff
@@ -0,0 +1,226 @@
+    });
+
+    it('should show an error in the input field if board size: 0', async () => {
+        const buttonDriver = ButtonTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_BUTTON,
+        });
+
+        await buttonDriver.click();
+        const inputDriver = InputWithLabelTestkit({ wrapper: wrapper, dataHook: BOARD_SIZE_INPUT });
+
+        await inputDriver.enterText('0');
+        expect(await inputDriver.getErrorMessage()).toEqual(BOARD_SIZE_MORE_THAN_ZERO);
+    });
+
+    it('should show "Player1\'s turn" after creating a game with boardSize: 2', async () => {
+        const buttonDriver = ButtonTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_BUTTON,
+        });
+
+        await buttonDriver.click();
+        const inputDriver = InputWithLabelTestkit({ wrapper: wrapper, dataHook: BOARD_SIZE_INPUT });
+
+        await inputDriver.enterText('2');
+        const messageBoxDriver = MessageBoxFunctionalLayoutTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_MESSAGE_BOX,
+        });
+        await messageBoxDriver.clickOnConfirmationButton();
+        const turnOrderHeadingDriver = HeadingTestkit({ wrapper, dataHook: TURN_ORDER_HEADING });
+        expect(await turnOrderHeadingDriver.exists()).toBeTruthy();
+        expect(await turnOrderHeadingDriver.getText()).toEqual("Player1's turn");
+    });
+
+    it('should show "Reset game" button after creating a game with boardSize: 2', async () => {
+        const createGameButtonDriver = ButtonTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_BUTTON,
+        });
+
+        await createGameButtonDriver.click();
+        const inputDriver = InputWithLabelTestkit({ wrapper: wrapper, dataHook: BOARD_SIZE_INPUT });
+
+        await inputDriver.enterText('2');
+        const messageBoxDriver = MessageBoxFunctionalLayoutTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_MESSAGE_BOX,
+        });
+        await messageBoxDriver.clickOnConfirmationButton();
+
+        const resetGameButtonDriver = ButtonTestkit({
+            wrapper: wrapper,
+            dataHook: RESET_GAME_BUTTON,
+        });
+        expect(await resetGameButtonDriver.exists()).toBeTruthy();
+    });
+
+    it('should show 4 squares after creating a game with boardSize: 2', async () => {
+        const createGameButtonDriver = ButtonTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_BUTTON,
+        });
+
+        await createGameButtonDriver.click();
+        const inputDriver = InputWithLabelTestkit({ wrapper: wrapper, dataHook: BOARD_SIZE_INPUT });
+
+        await inputDriver.enterText('2');
+        const messageBoxDriver = MessageBoxFunctionalLayoutTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_MESSAGE_BOX,
+        });
+        await messageBoxDriver.clickOnConfirmationButton();
+
+        const square0 = document.getElementById('0');
```

Comment:
'0' is not a good ID, there should be prefix before it

Reply:
I'll either need to refactor square implementation, because I rely on these number id's for the game logic, I could have some additional attribute for the square div...
But I'll rather change the test implementation so I won't need to query by id. I'm getting all of the squares by aria-label, then I `squares[0].click()`